src/deploy/osp_deployer/settings/sample_csp_profile.ini

```diff
@@ -687,11 +697,16 @@ powerflex_mgmt_interface=vlan881
 powerflex_cluster_interface=vlan170
 
 # Virtual IP from which the cluster is available.
-powerflex_cluster_virtualip=192.168.170.40
+powerflex_cluster_vip=192.168.170.40
+
+# By default, All traffic including SDS-SDS and SDS-SDC goes to the same network
+powerflex_rebuild_interface=vlan170
+
+# Comma separated list of disks to add to the SDSs. Must be the same name across all SDSs.
+powerflex_disks=['/dev/nvme0n1','/dev/nvme1n1','/dev/nvme2n1','/dev/nvme3n1','/dev/nvme4n1','/dev/nvme5n1','/dev/nvme6n1','/dev/nvme7n1']
 
```

Comment:
This looks like bad news to me. Ok for now but let s make a note / create a story to generate those on the fly

Reply:
It'll be enhanced but this is it for now.